kpops/components/streams_bootstrap/streams/model.py

```diff
@@ -281,6 +281,42 @@ class PrometheusJMXExporterConfig(CamelCaseConfigModel, DescConfigModel):
     )
 
 
+class JMXRule(SerializeAsOptionalModel, CamelCaseConfigModel, DescConfigModel):
```

Comment:
This seems to be the schema https://github.com/prometheus/jmx_exporter/blob/7e57a0807c39c05355194e84f51b4fb8719b21b7/collector/src/main/java/io/prometheus/jmx/JmxCollector.java#L73

Reply:
labelNames and labelValues is parsed from labels https://github.com/prometheus/jmx_exporter/blob/7e57a0807c39c05355194e84f51b4fb8719b21b7/collector/src/main/java/io/prometheus/jmx/JmxCollector.java#L452